treeherder/webapp/api/runnable_jobs.py

```diff
@@ -19,16 +21,17 @@ def list(self, request, project):
         """
         if 'decisionTaskID' in request.query_params and len(request.query_params['decisionTaskID']):
             decisionTaskID = request.query_params['decisionTaskID']
-            tc_jobs_url = "https://queue.taskcluster.net/v1/task/" + decisionTaskID + "/artifacts/public/full-task-graph.json"
+            tc_jobs_url = settings.TASKCLUSTER_FULL_TASK.replace("{{decision_task}}", decisionTaskID)
```

Comment:
Python has in-built string formatting, which is preferred over using `.replace()`. I'd also use a setting name of `TASKCLUSTER_TASKGRAPH_URL`, for consistency with eg `ALLTHETHINGS_URL`.

In settings.py use:
```python
TASKCLUSTER_TASKGRAPH_URL = 'https://queue.taskcluster.net/v1/task/{task_id}/artifacts/public/full-task-graph.json'
```
(Perhaps put it next to `ALLTHETHINGS_URL`, rather than at the end of settings.py, seeing as they are related?)

And then in runnable_jobs.py, use:
```python
tc_jobs_url = settings.TASKCLUSTER_TASKGRAPH_URL.format(task_id=decision_task_id)
```